dags/utils/gcp.py

```diff
@@ -221,7 +221,8 @@ def bigquery_etl_query(
     kwargs["name"] = kwargs.get("name", kwargs["task_id"].replace("_", "-"))
     if not project_id:
         project_id = "moz-fx-data-shared-prod"
-    sql_file_path = sql_file_path or "sql/{}/{}/{}/query.sql".format(project_id, dataset_id, destination_table)
+    destination_table_no_partition = destination_table.split("$")[0]
+    sql_file_path = sql_file_path or "sql/{}/{}/{}/query.sql".format(project_id, dataset_id, destination_table_no_partition)
```

Comment:
This should address https://github.com/mozilla/bigquery-etl/pull/3069/files/443cd7857e6c55fdbc0214bc15a315f073a2cacd#r909105233